src/curl.c

```diff
@@ -244,19 +244,32 @@ int swupd_curl_init(char *url)
 exit:
 	if (ret != 0) {
 		/* curl failed to initialize */
-		error("Failed to connect to update server: %s\n", globals.version_url);
+		error("Failed to connect to update server: %s\n", url);
```

Comment:
I think that this error message is misleading because the URL value is not always an update server. For instance, if I attempt to download a manifest tarball w/o network, I get the following error:

`Error: Failed to connect to update server: https://contenturl/10/Manifest.os-core.tar`

I would change to the following:

`error("Failed to connect to: %s\n", url);`